app/assets/stylesheets/topbar.scss

```diff
@@ -15,16 +22,27 @@
     margin-top: 6px;
   }
 
+  .top-bar-right {
+    margin-top: 15px;
+  }
 
-  .menu-text {
-    margin-top: 1px;
-    a {
-      font-size: 1.2rem;
+  .menu {
+    background-color: transparent;
+    margin-right: 10px;
+    li a {
+      padding-right: 20px;
     }
   }
 
-  .menu.vertical {
-    background-color: $rock;
+
+  .menu-text {
+    margin-top: 8px;
+    margin-left: 20px;
+    margin-bottom: 0px;
+    padding: 0px;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3